source/configuration_manual/replication.rst

```diff
@@ -0,0 +1,484 @@
+======================
+Replication with dsync
+======================
+
+.. note:
+
+  This is not supported with Dovecot Pro.
+
+Dovecot supports master/master replication using dsync. It's recommended
+that the same user always gets redirected to the same replica, but no
+changes get lost even if the same user modifies mails simultaneously on
+both replicas, some mails just might have to be redownloaded. The
+replication is done asynchronously, so high latency between the replicas
+isn't a problem. The replication is done by looking at Dovecot index
+files (not what exists in filesystem), so no mails get lost due to
+filesystem corruption or an accidental rm -rf, they will simply be
+replicated back.
+
+NOTE: v2.2 is highly recommended for this. Earlier versions can't do
+incremental metadata syncing. This means that the more mails a mailbox
+has, the slower it is to sync it.
+
+Replication works only between server pairs. If you have a large
+cluster, you need multiple independently functioning Dovecot backend
+pairs. This means that
+:ref:`director <dovecot_director>` isn't
+currently supported with replication. The replication in general is a
```

Comment:
Well, it could be handled with director tags.. Each server pair using a different tag.

Reply:
Yes, but no.